_data/team/roles/code_of_conduct.yml

```diff
@@ -0,0 +1,17 @@
+role: Code of conduct committee
+description: |
+  The [code of conduct committee](https://www.mdanalysis.org/pages/conduct/)
+  is elected by the MDAnalysis core developers on a rolling basis.
+tasks: 
+  - Act as points of contact for code of conduct violations
+
+current_leads:
+  - Jenna Swarthout Goddard
+  - Richard Gowers
+  - Micaela Matta
+current_members: []
+historical_members:
+  - Jenna Swarthout Goddard
+  - Richard Gowers
+  - Micaela Matta
```

Comment:
this isn't correct, but I'm also not sure that the previous CoC committee was public information either

Reply:
It wasn't. We had the CoCC for years (since we had the CoC) but I don't think it would be ok to put up names without first checking. It is a potentially sensitive issue. Maybe all we could say is "various (2017-2020)"

Reply:
Thanks for raising this! I'm just going to resolve the conversation here for now as we are not including historical members for now.